test/app/features/quiz/data/repositories/quiz_repository_test.dart

```diff
@@ -194,5 +212,86 @@ void main() {
         },
       );
     });
+
+    group('send', () {
```

Comment:
Aqui optei por fazer um teste próximo aos testes de integração, para ser possível validar o funcionamento do conjunto como um todo e facilitando quando for necessário remover código antigo.